main.go

```diff
@@ -207,6 +209,28 @@ func populateArgs(c *cli.Context) (
 	return
 }
 
+func callListRecursive(mountPoint string) (err error) {
+	logger.Debugf("Started recursive metadata-prefetch of directory: \"%s\" ...", mountPoint)
+	numItems := 0
+	err = filepath.WalkDir(mountPoint, func(path string, d fs.DirEntry, err error) error {
+		//logger.Infof("Walked path:%s, d=%s, isDir=%v", path, d.Name(), d.IsDir())
```

Comment:
Uncomment this and log at debug?

Reply:
It will be very chatty, even for debug logs. A bucket with million of objects will print out millions of log lines.

Reply:
Let's discuss this offline if needed.

Reply:
@kislaykishore ^

Reply:
> In that case, let's delete this comment: go/comments#comment_out

Yes. I will.